Update README.md (#1351)
rug-cit-hpc/league-of-robots: README.md

#### 3. Python virtual environment: install packages to manage Azure environment

> **! Make sure you already executed `ansible-galaxy install ... ` because it is providing the requirements.txt file !** (see previous step).

Reviewer: ```suggestion
**Make sure you already executed `ansible-galaxy install -r requirements.yml`** (see previous step), because it installs the `requirements.txt` file required by `pip` in this step.
```
